### PR TITLE
Added BasisCode Compliance Software to other.yml

### PR DESCRIPTION
**Pull request by baileybasiscode**
Added BasisCode Compliance Software because it has 2FA and contains private and secure information

### CODE REVIEW

**Review by Jawshy — CHANGES_REQUESTED** [review]
Thanks for submitting a pull request, @baileybasiscode. Could you please add an image for the site? The image will need to be 32 by 32 pixels, PNG format, and under 2,500 bytes in size. 

[Travis CI Build](https://travis-ci.org/2factorauth/twofactorauth/jobs/182165605)
```
BasisCode Compliance image not found.
```

File: _data/other.yml [hunk 1]
```diff
@@ -72,6 +72,14 @@ websites:
       tfa: No
       twitter: Azendoo
 
+    - name: BasisCode Compliance
+      url: http://basiscode.com
+      img: http://basiscode.com/wp-content/uploads/2016/10/BasisCode-Compliance.png
```
Comment (Jawshy) [inline _data/other.yml]: See [FAQ_New_TFA_SITE](https://github.com/2factorauth/twofactorauth/wiki/FAQ_New_TFA_SITE) for how to correctly add an image.

File: _data/other.yml [hunk 2]
```diff
@@ -72,6 +72,14 @@ websites:
...
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
```
Comment (Jawshy) [inline _data/other.yml:81]: Is there any public documentation for how two-factor authentication is implemented on the site?

**Comment by baileybasiscode** [comment]
Rich/all,

Thanks for the help and guidance. I believe I've figured it out (new to github). I've uploaded a new file (basiscode.png) to the img/other directory under the same branch (patch-3). 

About the text version of the png. I saw that I could create a new file, but not upload. So, I tried to create a new png file in the directory, then copy the "text" version of the file into the newly created file. Thanks for the comments as I was able to upload into the correct branch and submit.

Joe

**Comment by Carlgo11** [comment]
Hello Joe,
I'm afraid the logo you uploaded doesn't fit as it's 32x17 pixels instead of 32x32.
It's also not a very good image as it's [extremely hard to make out what it says](https://raw.githubusercontent.com/baileybasiscode/twofactorauth/262e52db6921ca689146c0ce7d8ea9084fbc0aa9/img/other/basiscode.png) when it's on such a small scale. Maybe you could find another image that better suits smaller sizes and preferably a squared one.

I can help you upload and shrink it if you prefer, however I need to know which image to use.
Just get back to me with a link or upload it using GitHub's drag and drop feature for comments.
index

//Carl

**Comment by baileybasiscode** [comment]
Carl/All,

Thanks for your patience. I played around and figure out how to create the graphic with the correct size. 

Thanks again!
Joe

**Comment by RichJeanes** [comment]
That's wonderful that you have figured it out (and welcome to GitHub!).

However, Carlgo11's and my comment still stand: That icon is not very well suited to being scaled to such a small size. Perhaps something in the ballpark of the site's favicon would be more appropriate?

**Comment by baileybasiscode** [comment]
Thanks for the feedback.. I'll work on it this evening and send something tomorrow. Again, thanks!

**Review by Jawshy — DISMISSED** [review]
> Is there any public documentation for how two-factor authentication is implemented on the site? 

See [FAQ_New_TFA_SITE](https://github.com/2factorauth/twofactorauth/wiki/FAQ_New_TFA_SITE).

**Review by Carlgo11 — DISMISSED** [review]
Needs a new logo as requested in [266583020](https://github.com/2factorauth/twofactorauth/pull/2223#issuecomment-266583020) and [266587168](https://github.com/2factorauth/twofactorauth/pull/2223#issuecomment-266587168).

**Comment by baileybasiscode** [comment]
Ok, I just submitted 2 pull requests on changeset 3 which includes a new graphic and a url to the 2FA documentation. Feel free to let me know if there are any issues. Thanks again. Joe

**Review by Jawshy — CHANGES_REQUESTED** [review]
> Ok, I just submitted 2 pull requests on changeset 3 which includes a new graphic and a url to the 2FA documentation. Feel free to let me know if there are any issues. Thanks again. Joe 

[Travis CI is having an issue](https://travis-ci.org/2factorauth/twofactorauth/jobs/185297262) with the [image that was used](https://github.com/baileybasiscode/twofactorauth/blob/9d5fab73bdb25294a6c846c20543704c517022c8/img/other/basiscode.png). Could you please replace it with the below image, @baileybasiscode? 

basiscode-resaved

**Comment by baileybasiscode** [comment]
Just pushed the latest update which includes the graphic. Thanks again for all of the help.
Joe

**Review by Jawshy — APPROVED** [review]

**Comment by baileybasiscode** [comment]
**You guys are the best! Thanks for all of your help! Joe**

**Pull request merged**